eng/pipelines/outerloop.yml

```diff
@@ -28,26 +28,29 @@ resources:
   - container: ubuntu_1604_arm_cross_container
     image: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-16.04-cross-14.04-23cacb0-20191023143847
 
+variables:
+  - template: variables.yml
+
 stages:
   # Windows outerloop legs
   - ${{ if or(endsWith(variables['Build.DefinitionName'], 'windows'), endsWith(variables['Build.DefinitionName'], 'outerloop')) }}:
-    - template: /eng/pipelines/windows.yml
+    - template: ${{ variables['pipelinesPath'] }}/windows.yml
       parameters:
         isOfficialBuild: ${{ and(ne(variables['System.TeamProject'], 'public'), notIn(variables['Build.Reason'], 'PullRequest')) }}
         fullMatrix: ${{ notIn(variables['Build.Reason'], 'PullRequest') }}
         testScope: outerloop
 
   # Linux outerloop legs
   - ${{ if or(endsWith(variables['Build.DefinitionName'], 'linux'), endsWith(variables['Build.DefinitionName'], 'outerloop')) }}:
-    - template: /eng/pipelines/linux.yml
+    - template: ${{ variables['pipelinesPath'] }}/linux.yml
```

Comment:
Is the full path here actually necessary?

Reply:
Actually not, just realized that now.

Don't review, will update some stuff.